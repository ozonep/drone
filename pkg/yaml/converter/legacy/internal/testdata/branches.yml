branches:
- master

pipeline:
  greeting:
    image: alpine
    commands:
    - echo hello
