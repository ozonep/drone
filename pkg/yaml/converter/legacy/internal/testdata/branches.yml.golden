---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: greeting
  pull: if-not-exists
  image: alpine
  commands:
  - echo hello

trigger:
  ref:
  - refs/pull/**
  - refs/pull-requests/**
  - refs/merge-requests/**
  - refs/heads/master

...
